root/etc/s6-overlay/s6-rc.d/svc-ddclient/run

```diff
@@ -2,4 +2,4 @@
 # shellcheck shell=bash
 
 exec \
-    s6-setuidgid abc /usr/bin/ddclient -foreground -file /ddclient.conf
+    s6-setuidgid abc /usr/bin/ddclient --foreground --file /config/ddclient.conf --cache /run/ddclient-cache
```

Comment:
This move warrants a changelog entry

Reply:
My only concern is that it was invisble to users before, so adding a changelog entry saying we moved the config file might confuse them.

Reply:
Hopefully [`0177e45` (#84)](https://github.com/linuxserver/docker-ddclient/pull/84/commits/0177e45f3381b0bb58a24d771a22c67ee4736296) covers it without confusing people.